ODCodeGen/CatOD2_CodeGen/CatOD2.consumption.cs

```diff
@@ -38,16 +38,6 @@ public class ModelInput
         #region model output class
         public class ModelOutput
         {
-            [ColumnName(@"Labels")]
-            public uint[] Labels { get; set; }
-
-            [ColumnName(@"Image")]
-            [Microsoft.ML.Transforms.Image.ImageType(600, 800)]
-            public MLImage Image { get; set; }
-
-            [ColumnName(@"Box")]
-            public float[] Box { get; set; }
-
```

Comment:
We don't need these for the ModelOutput

Reply:
They can be auto-gened from model output schema though

Reply:
Ah - is that why they're there?

Reply:
Looks like so

Reply:
Kinda noisy but I'm fine with it ... if that's why